packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -138,18 +138,24 @@ export const styles = css`
         flex-direction: column;
         overflow-y: auto;
         width: 100%;
+        --ni-private-listbox-num-visible-options: 10.5;
```

Comment:
We don't tend to use abbreviations, such as "num" here. What would you think about _--ni-private-listbox-visible-options-count_ instead?